{"name": "test_login_fail", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <web.testcases.test_login.TestLogin object at 0x7fafcf26aeb0>\nchrome_driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"750079ffc920fa45a5e2f7185404e427\")>\n\n    @pytest.mark.login\n    @allure.feature(\"登录失败\")\n    @allure.description(\"登录失败测试\")\n    def test_login_fail(self, chrome_driver):\n        \"\"\"\n        使用无法登录的用户\n        :param chrome_driver:\n        :param user:\n        :return:\n        \"\"\"\n        with allure.step(\"登录失败测试\"):\n            data = GetYamlConf(\"testdata.yaml\").env[\"prod_users\"][\"bad_user\"]\n            IndexPage().goto_loginPage(chrome_driver)\n>           LoginPage().login_submit(chrome_driver, data[\"username\"], data[\"password\"])\n\ntest_login.py:48: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../web_page/login_page.py:65: in login_submit\n    self.assert_login_success(driver)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <web.web_page.login_page.LoginPage object at 0x7fafcf26af70>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"750079ffc920fa45a5e2f7185404e427\")>\n\n    def assert_login_success(self, driver):\n        \"\"\"\n        断言是否登录成功\n        :param driver:\n        :return: 页面上是否出现发布订单按钮\n        \"\"\"\n        pub_order_button = self.puborder_btn_locator()\n        log.info(\"进行登录断言，断言结果下回再见\")\n>       assert self.find(driver, \"xpath\", pub_order_button)\nE       AssertionError\n\n../web_page/login_page.py:52: AssertionError"}, "description": "登录失败测试", "steps": [{"name": "登录失败测试", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/Users/cathy/PycharmProjects/HogwartsSDET15/web/testcases/test_login.py\", line 48, in test_login_fail\n    LoginPage().login_submit(chrome_driver, data[\"username\"], data[\"password\"])\n  File \"/Users/cathy/PycharmProjects/HogwartsSDET15/web/web_page/login_page.py\", line 65, in login_submit\n    self.assert_login_success(driver)\n  File \"/Users/cathy/PycharmProjects/HogwartsSDET15/web/web_page/login_page.py\", line 52, in assert_login_success\n    assert self.find(driver, \"xpath\", pub_order_button)\n"}, "start": 1701921105442, "stop": 1701921142555}], "attachments": [{"name": "失败截图.png", "source": "d0cf5e3c-c9a4-4d5d-a96b-f9141469d554-attachment.png", "type": "image/png"}, {"name": "log", "source": "621a44c3-b366-489f-8147-42881624cb63-attachment.txt", "type": "text/plain"}], "start": 1701921105442, "stop": 1701921142559, "uuid": "259f29d6-150d-4297-8dab-6c30dc829482", "historyId": "acf05160925847254b4e57a286be972e", "testCaseId": "acf05160925847254b4e57a286be972e", "fullName": "web.testcases.test_login.TestLogin#test_login_fail", "labels": [{"name": "epic", "value": "登录测试"}, {"name": "feature", "value": "登录失败"}, {"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "web.testcases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "cathydeMacBook-Air.local"}, {"name": "thread", "value": "33051-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "web.testcases.test_login"}]}